README.md

```diff
@@ -5,33 +5,42 @@
 [![Docker Pulls](https://img.shields.io/docker/pulls/frameworkcontroller/frameworkcontroller.svg)](https://hub.docker.com/u/frameworkcontroller)
 [![License](https://img.shields.io/github/license/microsoft/frameworkcontroller.svg)](https://github.com/microsoft/frameworkcontroller/blob/master/LICENSE)
 
-As one standalone component of [Microsoft OpenPAI](https://github.com/microsoft/pai), FrameworkController is built to orchestrate all kinds of applications on [Kubernetes](https://kubernetes.io) by a single controller.
+As one standalone component of [Microsoft OpenPAI](https://github.com/microsoft/pai), FrameworkController (FC) is built to orchestrate all kinds of applications on [Kubernetes](https://kubernetes.io) by a single controller, especially for DeepLearning applications.
 
 These kinds of applications include but not limited to:
-- __Stateless and Stateful Service__ (Nginx, TensorFlow Serving, HBase, Kafka, etc)
-- __Stateless and Stateful Batch__ (KD-Tree Building, Batch Data Processing, etc)
-- __Any combination of above applications__ (Distributed TensorFlow Training, Stream Data Processing, etc)
+- __Stateless and Stateful Service__:
+  - DeepLearning Serving: [TensorFlow Serving](https://www.tensorflow.org/tfx/guide/serving), etc.
+  - Big Data Serving: HDFS, HBase, Kafka, Nginx, etc.
+- __Stateless and Stateful Batch__:
+  - DeepLearning MPI Training: [Horovod Training](https://horovod.readthedocs.io), etc.
+  - DeepLearning Elastic Training without Server: [PyTorch Elastic Training with shared etcd](https://pytorch.org/elastic), etc.
```

Comment:
does "shared etcd" mean using k8s etcd? it's not easy to understand

Reply:
The "shared etcd" is from https://pytorch.org/elastic/0.2.0/examples.html#multi-node .
Any suggestion to better short name?

Reply:
how about "with whole cluster shared etcd" and "with per-application dedicated etcd"?

Reply:
that's fine, I didn't find that in pytorch docs